setup.py

```diff
@@ -73,56 +75,74 @@ def _gen_grpc(self):
 
         proto_root_dir = root / 'azure_functions_worker' / 'protos'
         proto_src_dir = proto_root_dir / '_src' / 'src' / 'proto'
-        staging_root_dir = root / 'build' / 'protos'
+        build_dir = root / 'build'
+        staging_root_dir = build_dir / 'protos'
         staging_dir = (staging_root_dir
                        / 'azure_functions_worker' / 'protos')
-        build_dir = staging_dir / 'azure_functions_worker' / 'protos'
-        built_protos_dir = root / 'build' / 'built_protos'
-        built_proto_files_dir = (built_protos_dir
-                                 / 'azure_functions_worker' / 'protos')
+        built_protos_dir = build_dir / 'built_protos'
 
-        if os.path.exists(build_dir):
+        if os.path.exists(staging_root_dir):
             shutil.rmtree(build_dir)
 
         if os.path.exists(built_protos_dir):
             shutil.rmtree(built_protos_dir)
```

Comment:
Seems like these four lines can be combined together?
```python
if os.path.exists(build_dir):
    shutil.rmtree(build_dir)
```
Am I missing something here?